prolog/comment.pl

```diff
@@ -116,6 +120,8 @@
     call(SS),
     comment_body_rec(SS,EE).
 
+
+%% ============================ testcode ==============================
```

Comment:
I believe the tests will need to be a unit test box in order for them to be triggered by our CI system.

However, it would probably be better to move this into a module in the test directory, so that we can keep the implementation and library code separate from the test code.